src/FormItem.tsx

```diff
@@ -40,6 +40,15 @@ export const FormItem = <TFieldValues extends FieldValues = FieldValues>({
           isValidElement(child) &&
           cloneElement(child, {
             ...field,
+            //@ts-expect-error onChange type safe is not necessary here
+            onChange: (...params) => {
```

Comment:
May I ask why `...params` here. cuz if you use a single `value`, this onChange can be type-safe. Is there a component have more than one params in their `onChanage` prop?

Reply:
Ahh I think I know why. But I still think type-safe is necessary here. Let me think a way to improve this.

Reply:
Yes.

For example [Select](https://ant.design/components/select) component has two params, `value` and `option`, for this reason i use spread operator.

Reply:
ahh make sense. then np!